app/src/main/assets/chart.html

```diff
@@ -190,9 +164,8 @@
     {% endif %}
   {% endfor %}
 
-  $(document).ready(function() {
-    $('#grid').freezeHeader({top: true, left: true});
-  });
+  $('#grid').freezeHeader({top: true, left: true});
```

Comment:
One other thing: I know that this does not concern this pull request in particular but the line bellow:
```javascript
var data = {{dataCellsByConceptId | raw}};
```
Is generating a `Uncaught SyntaxError: Unexpected token ;`. This is because it's being rendered like this:
```javascript
var data = ;
```
I noticed that dataCellsByConceptId is not being used anywhere else in the project so it will always be empty.
Reading runTileScript and runChartRowScript functions I noticed that both expects *data* to be an array so a good fix would also be to use hard coded brackets to avoid syntax errors. Like this:
```javascript
var data = [{{dataCellsByConceptId | raw}}];
```